src/test/java/seedu/address/model/command/CommandInfoTest.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.model.command;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class CommandInfoTest {
+
+    @Test
+    public void constructor_validCommandInfo_creationSuccessful() {
+        // Setup
```

Comment:
don't think these comments are necessary, should limit comments that don't add value